python_modules/dagster/dagster_tests/definitions_tests/asset_reconciliation_tests/scenarios.py

```diff
@@ -309,63 +352,6 @@ def _assets(context):
 # The cases
 ######################################################################
 
-
-class FanInPartitionMapping(PartitionMapping):
```

Comment:
these caused some headache- they get ignored when building the ExternalAssetGraph. Replaced with the equivalent StaticPartitionMappings